### DISCUSSION TITLE
Wrong date format when exporting

### DISCUSSION THREAD

**Discussion by tomzohar1** [retorquere/zotero-better-bibtex#2170] [Q&A]
I have used the zotero chrome plugin to add the following news article to my zotero:
https://www.ynet.co.il/articles/0,7340,L-4785472,00.html

When I did that, it automatically put in the "Date" information the following:"2016-03-31T06:39:36.06636Z"

When I try manually to change this I can do it, but it keeps coming back to the long format probably due to some automatic syncing.

I export the zotero to a bib file using the Better BibTex Format. In the bib file it says:

```
@article{Gal2016,
title = {{Abortion Pill Was Approved for Use Three Months Past Conception}},
shorttitle = {{"בשורה לנשים"}},
author = {Gal, Itay},
year = {2016-03-31T06:39:36.06636Z},
journal = {Ynet},
abstract = {מינהל המזון והתרופות האמריקני החליט להאריך את התקופה המותרת לגלולת ההפלות, מ-49 ימים אחרי הפסקת הווסת, ל-70 יום אחרי הפסקת הווסת. המשמעות: ניתן יהיה באמצעותה לבצע הפלה גם בחודש השלישי להריון. מומחה ישראלי: "בשורה גדולה לנשים"},
langid = {hebrew},
file = {/Users/tomzohar/Zotero/storage/5HE9KIZX/0,7340,L-4785472,00.html}
}
```

When I cite it on latex, it shows the following: (Gal, 2016-03-31T06:39:36.06636Z).

How can I fix this date format without it keep changing it back?

**Comment by retorquere** [comment]
Please post a BBT debug log ID for that item.
